core/optaplanner-core-impl/src/main/java/org/optaplanner/core/config/heuristic/selector/move/MoveSelectorConfig.java

```diff
@@ -33,10 +35,20 @@
  */
 
 @XmlSeeAlso({
-        UnionMoveSelectorConfig.class, CartesianProductMoveSelectorConfig.class, ChangeMoveSelectorConfig.class,
-        SwapMoveSelectorConfig.class, PillarChangeMoveSelectorConfig.class, PillarSwapMoveSelectorConfig.class,
-        TailChainSwapMoveSelectorConfig.class, SubChainChangeMoveSelectorConfig.class, SubChainSwapMoveSelectorConfig.class,
-        MoveListFactoryConfig.class, MoveIteratorFactoryConfig.class })
+        CartesianProductMoveSelectorConfig.class,
+        ChangeMoveSelectorConfig.class,
+        MoveIteratorFactoryConfig.class,
+        MoveListFactoryConfig.class,
+        PillarChangeMoveSelectorConfig.class,
+        PillarSwapMoveSelectorConfig.class,
+        SubChainChangeMoveSelectorConfig.class,
+        SubChainSwapMoveSelectorConfig.class,
+        SubListChangeMoveSelectorConfig.class,
+        SubListSwapMoveSelectorConfig.class,
+        SwapMoveSelectorConfig.class,
+        TailChainSwapMoveSelectorConfig.class,
+        UnionMoveSelectorConfig.class
+})
```

Comment:
Nitpick: the order has been changed here to alphabetaical but in the @XmlElement references below it's still by importance (composite first, then change/swap, then pillars, then chains, then customs). I'd keep it consistent.